frontend/src/pages/viewing_open_applications/StudentApplicationTable.js

```diff
@@ -0,0 +1,62 @@
+import * as React from "react";
+import { Table, Button, Container } from "react-bootstrap";
+
+const StudentApplicationTable = () => {
+  return (
+        <Container className="mt-3">
+            <Table hover size="sm">
+                <thead>
+                    <tr>
+                        <th scope="col">Course Name</th>
+                        <th scope="col">CRN</th>
+                        <th scope="col">Professor Name</th>
+                        <th scope="col">Semester</th>
+                        <th scope="col"></th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td>CS 1332</td>
+                        <td>54321</td>
+                        <td>Richard Landry</td>
+                        <td>Spring 2023</td>
+                        <td>
+                        <Button>View</Button>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td>CS 2050</td>
+                        <td>12345</td>
+                        <td>Gerandy Brito</td>
+                        <td>Spring 2023</td>
+                        <td>
+                        <Button>View</Button>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td>CS 4641</td>
+                        <td>54321</td>
+                        <td>Richard Landry</td>
+                        <td>Spring 2023</td>
+                        <td>
+                        <Button>View</Button>
+                        </td>
+                        
+                    </tr>
+                    <tr>
+                        <td>CS 3311</td>
+                        <td>25299</td>
```

Comment:
u should put this data in an array and render it here via another function or `.map` call